.azure-pipelines/stages.yml

```diff
@@ -69,6 +69,9 @@ stages:
       cacheTestResults: ${{ parameters.cacheTestResults }}
       authGithub: $(GitHubPublicRepoOnlyAccessToken)
       authGCP: $(GcpServiceAccountKey)
+      authGPGPassphrase: $(MaintainerGPGKeyPassphrase)
+      authGPGKey: $(MaintainerGPGKey)
```

Comment:
I think this should be `MaintainerGPGKeySecureFile....`

Reply:
arggh!!

updating 8/